zk/src/archive/web/js/zk/au.js

```diff
@@ -1145,6 +1145,42 @@ zAu.beforeSend = function (uri, req, dt) {
 		for (var errs = _onErrs.$clone(), fn; fn = errs.shift();)
 			if (fn(response, errCode))
 				return true; //ignored
+	},
+	/** Skip focus the busy widget, send focus to prev/next focusable element.
+	 * if widget prev/next focusable element is null, focus last/first focusable element.
+	 */
+	skipBusyWidget: function (e) {
+		e.preventDefault();
+		var focusableElements = '[href], :input,[tabindex]',
+			isTabKey = e.key == zKeys.TAB,
+			elem;
+		if (isTabKey && e.shiftKey) {
+			elem = zAu._getFirstFocusableElement(jq(this).prevAll(focusableElements));
+		} else if (isTabKey) {
+			elem = zAu._getFirstFocusableElement(jq(this).nextAll(focusableElements));
+		}
```

Comment:
//if widget prev/next focusable element is null, focus last/first focusable element.
only tab. shifTab preventDefault